### PR TITLE
add answers in dark mode

### PR DESCRIPTION
**Pull request by Marianna-Ikonomou**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,16 +23,17 @@ <h2><strong>Quiz App</strong></h2>
...
         <ul style="list-style: none;">
         <li>266</li>
-        <li>206</li>
+        <li data-js="correct-answer1">206</li>
         <li>316</li>
       </ul>
       </section>
       
-      <button>Show Answer</button>
+      <button type="button1" data-js="show-answer1">Show Answer</button>
```
Comment (sahedw) [inline index.html:31]: I don't know if the addition of "1" on the button type is necessary. As far as I know it only describes if its a submit, reset button etc.

**Comment by sahedw** [comment]
lgtm

File: index.html (hunk 1 above)
Comment (mpagels) [reply index.html:31]: This is a very good point. `button1` is indeed invalid.
Read more about the values you can pass to the `type` attribute [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type)

**Review by mpagels — COMMENTED** [review]
Just added a little note to the `button1` value to the `type` attribute. Highly recommend to check it out and try to fix this in a next pr. (did not check others prs)

That said: Very, very good work here. Keep it up. You're doing great!!

**Pull request merged**